app/serializers/export/post_serializer.rb

```diff
@@ -0,0 +1,16 @@
+module Export
+  class PostSerializer < ActiveModel::Serializer
+    attributes :text,
+               :author_id,
+               :public,
+               :diaspora_handle,
+               :type,
+               :image_url,
+               :image_height,
+               :image_width,
+               :likes_count,
+               :comments_count,
+               :reshares_count,
```

Comment:
I think it could be valuable for some - imagine you want to look at some old export of your data from an old pod for example. Maybe someone will make an app to analyse exports and rank posts with most likes etc.